testing/test_pdb.py

```diff
@@ -1,11 +1,17 @@
 from __future__ import absolute_import, division, print_function
 import sys
 import platform
+import os
 
 import _pytest._code
+from _pytest.debugging import (SUPPORTS_BREAKPOINT_BUILTIN, pytestPDB,
+                               pytest_configure)
 import pytest
 
 
+_ENVIRON_PYTHONBREAKPOINT = getattr(os.environ, 'PYTHONBREAKPOINT', '')
```

Comment:
Again this seems like it should be `_ENVIRON_PYTHONBREAKPOINT = os.environ.get('PYTHONBREAKPOINT', '')

Reply:
Fixed in f1f4c8c1